cypress/support/pages/signIn.pageObject.js

```diff
@@ -29,6 +37,16 @@ class SignInPageObject extends PageObject {
     this.signInBtn
       .click();
   }
+
+  verifyEmailErorr() {
```

Comment:
There is a typo in the method name `verifyEmailErorr`. It should likely be `verifyEmailError` to correctly reflect the purpose of verifying an email error.